keywording/text.xml

```diff
@@ -218,9 +218,9 @@ available.
 
 <p>
 If there are no new dependencies, do not remove keywords if your commit fails
-with repoman <d/> please try a full <c>git pull</c> and if you still have
-problems, then commit with <c>repoman -I</c> and file a bug to the broken
-architecture, noting it in your git commit message.
+with <c>repoman</c> or <c>pkgdev</c> <d/> please try a full <c>git pull</c> and
+if you still have problems, then commit with <c>repoman -I</c> and file a bug to
+the broken architecture, noting it in your git commit message.
```

Comment:
Sorry, I just noticed this wording "a full `git pull`" now. It raises the question if there are also non-full pulls? :smile: Checking the (somewhat convoluted) history shows that this used to say "a full `cvs update`" and was changed to the current wording in commit f59da125396e757847130b365361786e2ebec685.

With CVS that wording made sense, because a full update took a long time, and therefore a common workflow was to do updates only for a subdirectory (e.g., a package or a category). Which of course meant that your tree could be in an inconsistent state. A `cvs update` of the full tree would rectify that.

So, I think that mentioning this no longer makes sense today:
```suggestion
with <c>repoman</c> or <c>pkgdev</c> <d/> please commit with <c>repoman -I</c>
and file a bug to the broken architecture, noting it in your git commit message.
```

Reply:
Oh, that's really interesting. I hadn't thought about that part. Thank you!